speed-dreams/source-2.2.3/src/libs/tgfclient/guimenu.h

```diff
@@ -55,6 +55,13 @@ extern bool gfuiMenuGetBoolean(const char* pszValue, bool bDefault = false);
 #define GFMNU_TYPE_SCROLL_LIST "scroll list"
 #define GFMNU_TYPE_PROGRESS_BAR "progress bar"
 
+// SIMULATED DRIVING ASSISTANCE CHANGE: added radiobuttons type and attributes for it
+#define GFMNU_TYPE_RADIO_BUTTONS  "radio buttons"
```

Comment:
Rename to keep naming consistent throughout the code
```suggestion
#define GFMNU_TYPE_RADIO_BUTTON_LIST  "radio button list"
```